.changeset/tall-zoos-stare.md

```diff
@@ -0,0 +1,6 @@
+---
+'firebase': minor
+'@firebase/vertexai': minor
+---
+
+Add support for the Gemini Developer API, enabling usage in a free tier, and add new `AI` API to accomodate new product naming.
```

Comment:
Note that this isn't the *final* changelog that will be in the release notes. This will be changed to the final changelog entry we decide on when we write the release notes.